### PR TITLE
Add ai-lab chatbot helm chart OWNERS file

### PR DESCRIPTION
**Pull request by thepetk**
The PR is following the onboarding documentation for the openshift charts ([here](https://github.com/openshift-helm-charts/charts/tree/main/docs#submitting-chart-related-changes)).

As a result this is the first of the two PRs adding only the OWNERS of the community/ai-lab/chatbot/ chart.

After this one is merged a second PR will add the tar file of the chart.

### CODE REVIEW

File: charts/community/ai-lab/chatbot/OWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+chart:
+  name: chatbot-ai-software-template
```
Comment (elsony) [inline charts/community/ai-lab/chatbot/OWNERS:2]: Should we call this chatbot-ai-sample? Software template is a RHDH specific term.

File: charts/community/ai-lab/chatbot/OWNERS [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  shortDescription: Chatbot AI Software Template
```
Comment (elsony) [inline charts/community/ai-lab/chatbot/OWNERS:3]: See above

File: charts/community/ai-lab/chatbot/OWNERS (hunk 1 above)
Comment (thepetk) [reply charts/community/ai-lab/chatbot/OWNERS]: Addressed in [4b440cf](https://github.com/openshift-helm-charts/charts/pull/1669/commits/4b440cf74bdb485cfffdca12d1358e72d661e29a)

Comment (thepetk) [reply charts/community/ai-lab/chatbot/OWNERS]: I'll also update the helm chart package to be consistent with the name in the `Chart.yaml`

File: charts/community/ai-lab/chatbot/OWNERS (hunk 2 above)
Comment (thepetk) [reply charts/community/ai-lab/chatbot/OWNERS]: Addressed in [4b440cf](https://github.com/openshift-helm-charts/charts/pull/1669/commits/4b440cf74bdb485cfffdca12d1358e72d661e29a)

**Review by elsony — APPROVED** [review]
/lgtm

**Comment by thepetk** [comment]
> [INFO] OWNERS file changes require manual review by maintainers.

Looking into older PRs I think the error is because the check is related with the chart addition.

**Review by komish — APPROVED** [review]
/lgtm

**Pull request merged**